docs/content/preview/drivers-orms/python/django.md

```diff
@@ -158,47 +182,47 @@ urlpatterns = [
 ]
 ```
 
-This completes the configuration of your test application. The next steps are to create the migration files and apply the migrations to the database. To do this, run the following command:
+This completes the configuration of your test application. The next steps create the migration files and apply the migrations to the database. To do this, run the following command:
 
 ```shell
 python3 manage.py makemigrations
 python3 manage.py migrate
 ```
 
-A users table should be created in your database. Use the ysqlsh client shell to verify that the users table has been created in your database.
+A users table should be created in your database. Use the ysqlsh client shell to verify that the users table has been created.
 
 ### Run the application
 
 To run the application and insert a new row, execute the following steps.
 
-Run the django project using the following command:
+1. Run the django project using the following command:
 
-```shell
-python3 manage.py runserver 8080
-```
+    ```shell
+    python3 manage.py runserver 8080
+    ```
 
-Insert a row using the following command:
+1. Insert a row using the following command:
 
-```shell
-$ curl --data '{ "firstName" : "John", "lastName" : "Smith", "email" : "jsmith@yb.com" }' \
-       -v -X POST -H 'Content-Type:application/json' http://localhost:8080/users
-```
+    ```shell
+    $ curl --data '{ "firstName" : "John", "lastName" : "Smith", "email" : "jsmith@yb.com" }' \
```

Comment:
I think we are supposed to remove $ and such because when the reader does copy-paste, they get double shell prompt